### DISCUSSION TITLE
[Feature] adding labels

### DISCUSSION THREAD

**Discussion by JesseWebDotCom** [immich-app/immich#4880] [Feature Request]
### The feature

How about adding the ability to add labels to things like people, photos, albums?  My immediate need is to be able to label people - So for example, I could label some people as "mike" and "Walmart" and be able to quickly find / view Mike's coworkers when he worked at Walmart.  or "Tina" and "high school" to allow Tina to quickly see her high school friends. 

### Platform

- [ ] Server
- [ ] Web
- [ ] Mobile

**Comment by alextran1502** [comment]
duplicated of #3518
